### PR TITLE
fix releaser-tool error for using on  windows 

### PR DESCRIPTION
**Pull request by jjiill888**
1.  My error is that i  should use start.bat to create executable file not ps1.
2. I provide a win-pkg-example for using on windows,  exist in doc/

### CODE REVIEW

**Comment by jjiill888** [comment]
I found the highest version of java running for the current project is jdk17.  and I've tested it on my branch.

**Comment by jjiill888** [comment]
I found the highest version of java running for the current project is jdk17.  and I've tested it on my branch.

**Review by jjiill888 — COMMENTED** [review]
give a example for using on windows

File: README.zh_CN.md [hunk 1]
```diff
@@ -281,7 +281,7 @@ sbt:live2dforscala> exampleSWTLinux/assembly
 
 ### 7. 使用release-tool工具
 
-[警告]只支持linux，windows 运行时需要更改命令, 但是我们给了例子[example](doc/win-pkg-example.txt)
+[警告]只支持linux，但是我们给了windows使用的例子[example](doc/win-pkg-example.txt)
```
Comment (jjiill888) [inline README.zh_CN.md:284]: change:[警告]只支持linux，但是我们给了windows使用的例子[example](doc/win-pkg-example.txt)

**Comment by brianhsu** [comment]
Thanks, but I need to fix CI pipeline first during my free time before I can merge this. So it might take a little extra time, please be patient. Thanks.

**Comment by jjiill888** [comment]
I fix this problem in latest request, my fix comes from faceTracking.

 if you want to using swing version in windows, this fix causes an anomaly in graphics alignment in windows environments ╮(╯▽╰)╭ ╮(╯▽╰)╭ ╮(╯▽╰)╭

**Comment by brianhsu** [comment]
Sorry, but as stated in https://github.com/brianhsu/Live2DForScala/pull/5, I need to fix CI pipeline first (as you can see, the unit test failed after 360m, which is not normal and not acceptable). So I wouldn't merge this PR in a short time, I also have to revert your previous PR to verify what cause this.

I also need to  consider a better workflow for accept PR before I can accept any PR. The master should contains the latest release code in the original branching strategy, so merge PR into this branch will bring some problems to the CI pipeline.

I need to figure out this first. Temporary close this PR.

**Comment by jjiill888** [comment]
I'm sorry I got you in trouble.The unit test failure was really something that shouldn't have happened, I will wait for your latest stable update and use this as my fork base afterward!

 I've never touched unit testing before, I should have tested it before submitting the pr, I'm not quite sure what i need to do to submit stable code, may i need youtube's online class.

**Pull request closed without merging**